src/main/java/winstone/AbstractSecuredConnectorFactory.java

```diff
@@ -197,14 +199,20 @@ protected SslContextFactory getSSLContext( Map args) {
             }
 
             SslContextFactory ssl = new SslContextFactory();
-
             ssl.setKeyStore(keystore);
             ssl.setKeyStorePassword(keystorePassword);
             ssl.setKeyManagerPassword(privateKeyPassword);
             ssl.setKeyManagerFactoryAlgorithm(Option.HTTPS_KEY_MANAGER_TYPE.get(args));
             ssl.setCertAlias(Option.HTTPS_CERTIFICATE_ALIAS.get(args));
             ssl.setExcludeProtocols("SSLv3", "SSLv2", "SSLv2Hello");
-
+            String excludeCiphers = Option.HTTPS_EXCLUDE_CIPHER_SUITES.get( args );
+            if( StringUtil.isNotBlank(excludeCiphers)) {
```

Comment:
I think it worth mentioning in the description of the option how to provide "no exclusion". Passing an empty string will let the default exclusion in place.

Reply:
good point!
the no exclusion is currently a fake exclusion such ` --excludeCipherSuites=BEER` 
maybe we can let users pass an empty string if they want no exclusion? (using isNotEmpty rather than isNotBlank)
WDYT?

Reply:
I would prefer such approach than the random string :) Imagine the admin that should maintain such configuration, good luck to find why the previous admin put a random string (that is not random for him)

Reply:
done possible to exclude NO cipher by using ` --excludeCipherSuites=' ' `